services/ui/ws/platform_display.h

```diff
@@ -61,6 +61,10 @@ class PlatformDisplay : public ui::EventSource {
 
   virtual FrameGenerator* GetFrameGenerator() = 0;
 
+  // Shows or hides native window.
+  virtual void Hide() {}
```

Comment:
These are the first occurrence of "Show" in services/ui/ws.

Maybe stick with the SetWindowVisibility-like nomenclature?